pkg/features/versioned_kube_features.go

```diff
@@ -31,4 +32,7 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 	RelaxedEnvironmentVariableValidation: {
 		{Version: version.MustParse("1.30"), Default: false, PreRelease: featuregate.Alpha},
 	},
+	genericfeatures.BtreeWatchCache: {
+		{Version: version.MustParse("1.31"), Default: true, PreRelease: featuregate.Beta},
```

Comment:
Yes, but this would deactivate the feature. Need to ask @liggitt if we will move emulated version to v1.32 so we can start testing features landing in v1.32.

Reply:
cc @Jefftree @aaron-prindle

Reply:
Opened a tracking issue https://github.com/kubernetes/kubernetes/issues/126946, and will work on getting 1.32 working.